README.md

```diff
@@ -325,6 +325,18 @@ service should be in a `running (ealthy)` state.
 
 If any service is in an `exited` or `unhealthy` state, investigate further. Be sure to refer to the [**Troubleshooting documentation**](./TROUBLESHOOTING.md#Enterprise-Control-Plane-and-Data-Plane-Connectivity-Issues)
 
+Open up a browser window and navigate to Kong Manager using following address:
```

Comment:
We didn't not mention how to access Kong Manager. So add this block to add URLs of Kong Manager and the Admin API.